packages/workspace-utils/src/index.sql

```diff
@@ -118,3 +125,40 @@ VALUES
     ),
     $message
   );
+
+-- BLOCK update_course_instance_usages_for_workspace
+INSERT INTO
+  course_instance_usages (
+    type,
+    institution_id,
+    course_id,
+    course_instance_id,
+    date,
+    user_id,
+    include_in_statistics,
+    duration
+  )
+SELECT
+  'Workspace',
+  i.id,
+  c.id,
+  ci.id,
+  date_trunc('day', w.state_updated_at, 'UTC'),
+  v.authn_user_id,
```

Comment:
Using `state_updated_at` and `v.authn_user_id` again. I think this timestamp is ok?